tests/unittests/test_transaction_canonicalize.py

```diff
@@ -22,6 +22,12 @@ def test_lowercases_if_capital_only_exists(self):
         canonicalize(record, "foo")
         self.assertEqual(record, expected_record)
 
+    def test_null_receipt_record(self):
+        record = {"receipt": None}
+        expected_record = {"receipt": None}
+        canonicalize(record, "foo")
+        self.assertEqual(record, expected_record)
+
```

Comment:
Running this code against current master branch will yield a crash. Running it against this PR yields a success.